interface/patient_file/pos_checkout.php

```diff
@@ -164,87 +164,77 @@ function generate_receipt($patient_id, $encounter = 0)
     $encrow = sqlQuery("SELECT invoice_refno FROM form_encounter WHERE " .
     "pid = ? AND encounter = ? LIMIT 1", array($patient_id,$encounter));
     $invoice_refno = $encrow['invoice_refno'];
-
-    // being deliberately echoed to indicate it is part of the php function generate_receipt
-    echo "<!DOCTYPE html>" . PHP_EOL;
-    echo "<html>" . PHP_EOL;
-    echo"<head>" . PHP_EOL;
     ?>
+    <!-- being deliberately echoed to indicate it is part of the php function generate_receipt -->
```

Comment:
@stu01509 Are you trying to remove the generate_receipt function here? If not you need to revert what you did here with converting these echo statements.

Reply:
I didn't remove the generate_receipt here, I just wanna convert these echo statements, are not allow convert here?

Reply:
Yeah I wouldn't touch this area due to this comment being here